react/src/pages/ServingListPage.tsx

```diff
@@ -273,6 +273,26 @@ const ServingListPage: React.FC<PropsWithChildren> = ({ children }) => {
               name
               endpoint_id
               image
+              image_object @since(version: "23.09.9") {
```

Comment:
Thia page doesn't use `image` and `image_object`, right?. If yes, please remove these in this file.